src/hoodie/request.js

```diff
@@ -64,46 +66,48 @@ exports.request = function(hoodie, type, url, options) {
     url = (hoodie.baseUrl || '') + API_PATH + url;
   }
 
-  // if url is cross domain, set CORS headers
-  if (/^http/.test(url)) {
-    defaults.xhrFields = {
-      withCredentials: true
-    };
-    defaults.crossDomain = true;
-  }
-
   defaults.url = url;
+  
+  //passing `data` as an option is throughout the code
+  if(options.data){
+    options.body = options.data;
+    delete options.data;
+  }
 
-  // we are piping the result of the request to return a nicer
-  // error if the request cannot reach the server at all.
-  // We can't return the promise of ajax directly because of
-  // the piping, as for whatever reason the returned promise
-  // does not have the `abort` method any more, maybe others
-  // as well. See also http://bugs.jquery.com/ticket/14104
-  jQueryPromise = global.jQuery.ajax(extend(defaults, options))
-    .done(requestDefer.resolve)
-    .fail(requestDefer.reject);
-  var pipedPromise = requestPromise.catch(exports.handleRequestError.bind(null, hoodie));
-  pipedPromise.abort = jQueryPromise.abort;
-
-  return pipedPromise;
+  var ajaxRequest = ajax(extend(defaults, options), function(err, response, data){
+    if(err){
+      return requestDefer.reject(err);
+    }
+    requestDefer.resolve(response, data);
+  });
+  var promise = requestPromise.catch(exports.handleRequestError.bind(null, hoodie));
+  
+  promise.abort = ajaxRequest.abort;
+  return promise;
 };
 
 //
 //
 //
-exports.handleRequestError = function(hoodie, xhr) {
+exports.handleRequestError = function(hoodie, pouchError) {
   var error;
 
   // handle manual abort of request
-  if (xhr.statusText === 'abort') {
+  //I can't immediately see a way to port this
+  //pouchdb-ajax doesn't expose the xhr or it's error
+  //directly, and afaik it doesn't pass along any
+  //means to direct an abort
+  /*if (xhr.statusText === 'abort') {
 
     return rejectWith({
       name: 'HoodieConnectionAbortError',
       message: 'Request has been aborted',
     });
-  }
+  }*/
 
```

Comment:
See https://github.com/pouchdb/ajax/issues/1 

Since it's not currently being used by PouchDB proper, I'm not sure how battle-ready the `pouchdb-ajax` library is.  Might be a good idea to look into an alternative.  thoughts?